pages/content/amp-dev/documentation/guides-and-tutorials/integrate/story-interactive.md

```diff
@@ -0,0 +1,71 @@
+
+### Sizing
+
+The interactive elements work great across different screen resolutions by having a size that is dependent on the `font-size` value, which is by default set to a percentage of the screen size to adapt the elements correctly. However, story creators can resize the elements by changing the `font-size` rule. The recommended way of integrating sizing into editors is either to not allow users to change the size at all, or to give users a slider with good boundaries. For instance, if a tool wants to use `amp-story-grid-layer[aspect-ratio]` layouts, they might want to set a slider to go between 0.15ems and 0.3ems, which are good extremes for very large and very small elements. Tools can use ems, rems or percentage of `var(--story-page-vmin)` for instance, but is recommended to use the same type of units that they use for text, as these elements can be thought of as "regular text containers". Changing the `width` property should only be made within the predefined min and max values (if at all).
+
+### Content management
+
+Different tools have different degrees of content customization, but interactive elements have a fixed set of attributes that have to be specified in the elements for them to work properly. Tools should provide placeholders for 2 options and allow users to add up to 4 options when possible (either by editing the content inline or on a side menu), but should never allow a number of options that is not supported by the validation rules.
+
+Elements that require image URLs such as amp-story-interactive-results can use already existing workflows to upload and show images.
+
+Multi-state elements such as amp-story-interactive-results should allow users to change the values on each state independently either inline or on a side panel, but it's a good idea for users to preview each state as they are working on them, so having a preview selector could be a good idea.
+
+Previews cannot always visualize all the fields (such as confetti effects), but tools can show a preview of the animation only played when users change the values associated with it. Remember that some elements allow all options to have a confetti emoji, while others limit the confetti to certain options. 
+
+[tip type="note"]
+Adding animations to the previews can also allow `animate-in` effects in the interactive elements, which are a great way to customize the experience further.
+[/tip]
+
+Some elements have additional configuration for their options, which will require custom code in the UI to input (eg: what option is correct in a quiz), but most elements share to some extent all the content attributes, so reusing the code and UI is a good idea.
```

Comment:
This feels like we're trying to tell creation tools how to do their job rather than what's possible with the components and tips to integrate them.